src/features/account/components/AccountRoleTag.test.tsx

```diff
@@ -0,0 +1,38 @@
+import { AccountRoleTags, AccountRole } from './AccountRoleTags';
+import { render } from '../../../utils/test-utils';
+
+describe('AccountRoleTags', () => {
+  it('renders', () => {
+    const { queryByTestId } = render(
+      <AccountRoleTags
+        roles={[
+          AccountRole.AUDITOR,
+          AccountRole.DEVELOPER,
+          AccountRole.REVIEWER,
+        ]}
+      />,
+    );
+
+    expect(queryByTestId('account-role-developer')).toBeInTheDocument();
+    expect(queryByTestId('account-role-reviewer')).toBeInTheDocument();
+    expect(queryByTestId('account-role-auditor')).toBeInTheDocument();
+  });
+
+  it('does not render account role tags if provided `roles` is not an `AccountRole`', () => {
+    const { queryByTestId } = render(
+      <AccountRoleTags roles={['test' as unknown as AccountRole]} />,
```

Comment:
`@ts-expect-error` will result in a compiler error if the role is somehow valid, which can help catch regressions.
```suggestion
      // @ts-expect-error - Invalid role.
      <AccountRoleTags roles={['test']} />,
```

Reply:
noted

Reply:
updated